modules/samples/connectedcup/component/agent/src/main/webapp/index.jsp

```diff
@@ -1,140 +1,134 @@
-
-
-            }
-
-            function processTemperature(min, max){
-                $( "#temperature_level" ).slider({
-                    orientation: "vertical",
-                    range: "min",
-                    min: min,
-                    max: max,
-                    value: 0,
-                    slide: function( event, ui ) {
-                        $( "#amount_temp" ).val( ui.value );
-                    }
-                });
-            }
-
-            function map(a,b,c,d,N)
-            {
-                return c+((N-a)*(d-c))/(b-a);
-            }
-
-            processCoffeeLevel(0, 100);
-            processTemperature(0,100);
-
-            $("#order-cup").click(function() {
-
-                var deviceId = '<%=request.getSession().getAttribute("deviceId")%>';
-                var deviceOwner = '<%=request.getSession().getAttribute("deviceOwner")%>';
-                var token = '<%=request.getSession().getAttribute("token")%>';
-                var url = "https://localhost:9443/connectedcup/controller/ordercoffee?deviceId=" + deviceId +"&deviceOwner=" +
-                        deviceOwner;
-
-                $.ajax({
-                    type: 'POST',
-                    url: url,
-                    headers: {
-                        "Authorization" : "Bearer " + token
-
-                    }
-                });
-
-            });
-
-            function sendData()
-            {
-                var deviceId = '<%=request.getSession().getAttribute("deviceId")%>';
-                var deviceOwner = '<%=request.getSession().getAttribute("deviceOwner")%>';
-                var tempPayload = "temperature:" + $( "#temperature_level" ).slider( "value" );
-                var levelPayload = "coffeelevel:" + $( "#slider-vertical" ).slider( "value" );
-                $.post( "/connected-cup-agent/push_temperature?deviceId=" + deviceId +"&deviceOwner=" + deviceOwner +
-                        "&payload=" + tempPayload);
-                $.post( "/connected-cup-agent/push_level?deviceId=" + deviceId +"&deviceOwner=" + deviceOwner +
-                        "&payload=" + levelPayload);
-                setTimeout(sendData, 5000);
-            }
-
-            sendData();
-
-        </script>
-    </body>
-</html>
+</div>
+
+<script src="js/libs/jquery.min.js"></script>
+<script src="js/coffee.js"></script>
+<script src="js/libs/htmlpreview.min.js"></script>
+<script>HTMLPreview.replaceAssets();</script>
+<script>
+    $("#order-cup").click(function() {
+
+        var deviceId = '<%=request.getSession().getAttribute("deviceId")%>';
+        var deviceOwner = '<%=request.getSession().getAttribute("deviceOwner")%>';
+        var token = '<%=request.getSession().getAttribute("token")%>';
+        var url = "/connectedcup/controller/ordercoffee?deviceId=" + deviceId +"&deviceOwner=" +
+                  deviceOwner;
+
+        $.ajax({
+                   type: 'POST',
```

Comment:
Single quote is the standard. So need to change double quoted strings to single quoted. Fixed in PR https://github.com/wso2/product-iots/pull/179
https://google.github.io/styleguide/javascriptguide.xml?showone=Strings#Strings